src/app/news/news-list-item.component.ts

```diff
@@ -131,4 +134,12 @@ export class NewsListItemComponent implements OnChanges, AfterViewChecked {
       (!this.targetLocalPlanet || (!this.newsService.postSharedWithCommunity(this.item) && this.isMainPostShared));
   }
 
+  shouldShowIndication() {
+    if (this.shareTarget === this.planetCode) {
```

Comment:
Two things:
- Rather than `shareTarget` you should compare `this.planetCode` to `this.item.doc.user.planetCode`.
- The condition in the if can be combined with the `return` statement inside to make this a one line function that reads a little better.